pages/common/keep-header.md

```diff
@@ -0,0 +1,20 @@
+# keep-header
+
+> Convenience utility that runs Unix commands in a header-aware fashion.
```

Comment:
```suggestion
> Ignore the first line of a file when running a command.
```

Does this describe the command in an easier way in your opinion?

Reply:
What about:
`Keep first line untouched by a command, passing it directly to stdout`

Ignoring does not specify that the line ends up in stdout - could just not be passed through?